src/live_restore/live_restore_private.h

```diff
@@ -8,7 +8,7 @@
 
 #pragma once
 
-#define WTI_LIVE_RESTORE_FS_TOMBSTONE_SUFFIX ".deleted"
+#define WTI_LIVE_RESTORE_STOP_FILE_SUFFIX ".stop"
```

Comment:
I don't think we've documented what a stop file is anywhere. Either here or in the function comment for `__live_restore_create_stop_file_path` seems reasonable.

```
Stop files are created in the file system (for example "foo.wt" would have a stop file "foo.wt.stop") to indicate that a file has been completely migrated from the source to the destination. If a stop file exists we will never need to access the source file again.
```

Reply:
Rephrased it a bit: [a0b427f](https://github.com/wiredtiger/wiredtiger/pull/11507/commits/a0b427f7356c60d8e29fd916e92c266e5af58383)